pkg/daemons/agent/agent_windows.go

```diff
@@ -48,11 +49,9 @@ func kubeletArgsAndConfig(cfg *config.Agent) (map[string]string, *kubeletconfig.
 		return nil, nil, err
 	}
 	argsMap := map[string]string{
+		"config": filepath.Join(cfg.KubeletConfigDir, "00-"+version.Program+"-defaults.conf"),
```

Comment:
This will result in loading the config twice, first as the config, and then again as a drop-in. As discussed in the upstream issue, there are critical difference in behavior wrt defaults when the config file is loaded as config vs a config drop-in.

Reply:
it is currently "fixed" by using the CLI flag, as we previously did. If you're going to revert and rework an existing change it would be good to provide some justification as to why it is an improvement over the current behavior. 

Ideally this would be done in the appropriate place in our PR template - instead of just deleting the template and saying "fixes X" with a link to an already-closed issue.